src/main/java/com/sfuentes/Employee.java

```diff
@@ -0,0 +1,37 @@
+package com.sfuentes;
+
+public class Employee {
+
+  int employeeId;
+  final double baseSalary = 10000.00;
+  final double bonusPercentage = 10;
+  double[] listSales;
+  double netSalary;
+
+  public Employee(int employeeId, double[] listSales) {
+    this.employeeId = employeeId;
+
+    if (listSales.length == 3)
+      this.listSales = listSales;
+    else
+      throw new IllegalArgumentException("Incorrect amount of sales");
+  }
+
+  public double calculateSumSalary() {
+    double sumSalary = 0.0;
```

Comment:
Same here this is not the sum of the salary this variable will store the sum of the 3 sales

Reply:
fixed